### ISSUE TITLE
Question about the training pipeline

### ISSUE THREAD

**Issue by LZL-CS** [dichotomies/N3F#6]
Hi,
I dove into your code, and I found that you train your model based on a pre-trained NeRF model.  Can I train the whole model (add a feature head) from scratch without using the pre-trained NeRF model? Will this cause theperfomance of the model to degenerate?

Thanks!

**Comment by dichotomies** [comment]
You can also train the model from scratch, but you will have to fine-tune the hyperparameters. If I remember correctly, it's helpful to keep the weight of the feature loss lower and increase it over time. There is a [recent paper](https://github.com/lmur98/DIV_FF_CVPR) that is based on N3F, looking into this might be helpful as well.

**Comment by LZL-CS** [comment]
> recent paper

Thanks for your prompt reply and suggestion, I will check it!
